doc/Type/independent-routines.pod6

```diff
@@ -1164,6 +1164,65 @@ assigning L<Empty|/type/Slip#index-entry-Empty-Empty>; for everything else,
 equivalent to assigning L<Nil|/type/Nil> or C<Empty> in the case of arrays or
 hashes, whose use is advised.
 
+=head1 Array manipulation
+
+Routines that manipulate arrays and other containers.
+
+=head2 sub pop
+
+Defined as:
+
+    proto sub pop($, *%)
+    multi sub pop(@a)
+
+Calls method L<C<pop>|/routine/pop> on the container, which is supposed to
+remove and return the last element, or fail if the container is empty.
+
+=head2 sub shift
+
+Defined as:
+
+    proto sub shift($, *%)
+    multi sub shift(@a)
+
+Calls method L<C<shift>|/routine/shift> on the container which is supposed to
+remove and return the first element, or fail if the container is empty.
+
+=head2 sub push
+
+Defined as:
+
+    proto sub push($, |)
+    multi sub push(\a, \b)
+    multi sub push(\a, **@b)
+
+Probably best thought of as a simple:
```

Comment:
There's a `sub push` example in [`Hash`](https://docs.raku.org/type/Hash#method_push) which is not included in the definition. Maybe a reference to this definition would be needed, as well as an example of how to use it here.

Reply:
In [`Hash`](https://docs.raku.org/type/Hash#method_push) I will add a link to the subroutine documentation.

Reply:
You might want to move the example too.